patches/displayFullName.js

```diff
@@ -0,0 +1,15 @@
+let studentData = document.querySelector(".side_student");
+if (window.location.hostname.match(/(dziennik-)?wiadomosci.*/))
+  studentData = document.querySelector(".account__name span").firstChild;
+
+if (!studentData.textContent) return;
```

Comment:
![image](https://github.com/user-attachments/assets/e2192e32-200f-4b5a-92c3-c26532f3f3d9)
You can't use return here